plasmapy/classes/plasma.py

```diff
@@ -59,6 +59,7 @@ def __init__(self, domain_x, domain_y, domain_z, gamma=5/3):
 
         x, y, z = self.x.si.value, self.y.si.value, self.z.si.value
         self.grid = np.meshgrid(x, y, z, indexing='ij') * u.m
+        self.grid = np.squeeze(self.grid)
```

Comment:
You're right, we need to keep an eye on that. At the moment I'm just making sure that I use dimensions in the order x, y, z so that it doesn't break, but at some point I'll try and make it a bit more robust.